### ISSUE TITLE
Veyon causing computers to black screen

### ISSUE THREAD

**Issue by gavctech** [veyon/veyon#500]
I am running a lab of about 25 computers, and most of the time I do not have any problems. However, recently I have found that almost every day, the computers just go to a black screen for no reason. We usually have to restart the computer and that fixes it for a little bit, but then the problem happens again. Anyone know why this is happening?

**Comment by tobydox** [comment]
Is this related to Veyon? Which version of Veyon do you have running? Does the problem disappear on a computer when uninstalling Veyon or disabling the Veyon Service?

**Comment by caspio21** [comment]
Hi there I'm having the same problem on several machines. I have about 20 machines.
It all started after enabling multi-session in Veyon.
I removed it from 3 computers and the problem went away.
Since the CCP virus social distancing, I had to setup all the machines to allow multissession for "Microsoft RDP" for users to log-in remotely.

The way I set them up was:
Stopping veyon service from starting automatically (since I don't want to waste resources having 2 sessions [local console] + [remote session]). Only need the remote session
Then I have a user logon script that kills all existing "veyon-server.exe" processes, then assigned VEYON_SESSION_ID=15 system variable and then executes veyon-server.exe as the authenticated user.

The computers work for a day or two. Then, when a RDP session is initiated (after succesfull login) they get a black screen, then the RDP window just vanishes without error messages.

The local console is still responsive and I use it to reboot the machine. After rebbot everting works again for a day or two and the cycle repeats.
